test/integration/targets/inventory_foreman_script/runme.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+[[ -n "$DEBUG" || -n "$ANSIBLE_DEBUG" ]] && set -x
+
+set -euo pipefail
+
+export ANSIBLE_CONFIG=ansible.cfg
+export FOREMAN_HOST="${FOREMAN_HOST:-localhost}"
+export FOREMAN_PORT="${FOREMAN_PORT:-8080}"
+FOREMAN_CONFIG=foreman.ini
+
+# flag for checking whether cleanup has already fired
+_is_clean=
+
+function _cleanup() {
+    [[ -n "$_is_clean" ]] && return  # don't double-clean
+    echo Cleanup: removing $FOREMAN_CONFIG...
+    rm -vf "$FOREMAN_CONFIG"
+    echo Cleanup: removing foreman.py...
+    rm -v foreman.py
+    echo Cleanup: removing inventory.json...
+    rm -vf inventory.json
+    unset ANSIBLE_CONFIG
+    unset FOREMAN_HOST
+    unset FOREMAN_PORT
+    unset FOREMAN_CONFIG
```

Comment:
Environment variables won't survive the test, so there shouldn't be any reason to unset them. In fact, cleanup shouldn't be needed at all as long as everything is written only to the temporary directory `OUTPUT_DIR`.